backend/webhook/consumer/pkg/worker/worker.go

```diff
@@ -0,0 +1,135 @@
+package worker
+
+import (
+	"errors"
+
+	beanstalk "github.com/beanstalkd/go-beanstalk"
+
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	"log"
+	"net/http"
+	"time"
+)
+
+type ConsumerError struct {
+	Err error
+	T   time.Time
+}
+
+type Task struct {
+	done      chan bool
+	beanstalk *beanstalk.Conn
+	errors    []ConsumerError
+}
+
+type AiryMessage struct {
```

Comment:
WebhookEvent would reflect the naming we use in other places (even thought the data type is slighly different because of implementation detail. But better name than "message")